Project/Assets/Prairie/Framework/Script/Interaction/Annotation.cs

```diff
@@ -34,7 +34,7 @@ public class Annotation : Interaction
     public TextAsset textFile;
     public string text = "";
     public List<Texture2D> images;
-    public string summary = "";
+    public string summary;
 
```

Comment:
Why did you remove the default value for `summary`? This is probably here for a reason (to ensure summary is never null) - and your PR seems unrelated to the summary implementation.